### PR TITLE
Update release notes for v1.1.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

If we land anything else before cutting a tag we'll need to update more, so make sure the last few PRs are merged before we merge this.

### CODE REVIEW

**Comment by mheon** [comment]
Hm. Markdown doesn't like my links. Will touch up.

**Comment by vrothberg** [comment]
Great release notes, @mheon :+1:

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+- Fixed a bug where `podman login` would sometimes report it logged in successfully when it did not
+- Fixed a bug where `podman pod create` would not error on receiving unused CLI argument
+- Fixed a bug where rootless `podman run` with the `--pod` argument would fail if the pod was stopped
+- Fixed a bug where `podman images` did not print a trailing newline when not invoked on a TTY ([#2388](https://github.com/containers/libpod/issues/2388))
+- Fixed a bug where the `--runtime` option was sometimes not overriding `libpod.conf`
+- Fixed a bug where `podman pull` and `podman runlabel` would sometimes exit with 0 when they should have exited with an error ([#2405](https://github.com/containers/libpod/issues/2405))
+- Fixed a bug where rootless `podman export -o` would fail ([#2381](https://github.com/containers/libpod/issues/2381))
+
+### Misc
+- Podman migrated to a new, shared memory locking model in this release. As part of this, if you are running Podman with pods or dependency containers (e.g. `--net=container:`), you should run the `podman system renumber` command to migrate your containers to the new model
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: As this one has some warnings about when to run system renumber, is it worthwhile to add something like:  "Please reference the `podman-system-renumber(1)` man page for further details"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+- Podman migrated to a new command-line parsing library, and the output format of help and usage text has somewhat changed as a result
+- Updated vendored Buildah to v1.7, picking up a number of bugfixes
+- Updated vendored containers/image library to v1.4, picking up a number of bugfixes and performance improvements to pushing images
+- Updated vendored containers/storage library to v1.10, picking up a number of bugfixes
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: "Updated vendored" sounds odd.  Could we just go with "Vendored" for lines like this?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+- Work on the remote Podman client for interacting with Podman remotely over Varlink is progressing steadily, and many image and pod commands are supported
+- Added path masking to mounts with the `:z` and `:Z` options, preventing users from accidentally performing an SELinux relabel of their entire home directory
+- The `podman container runlabel` command will not pull an image if it does not contain the requested label
+- Many commands' usage information now includes examples
+- `podman rm` can now delete containers in containers/storage, which can be used to resolve some situations where Podman fails to remove a container
+- The `podman search` command now searches multiple registries in parallel for improved performance
+- The `podman build` command now defaults `--pull-always` to true
+
 ## 1.0.0
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: I think you should mention the conversion to Cobra in here somewhere.  I'd suggest Features, but at least in Misc.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,68 @@
...
+- Updated vendored containers/image library to v1.4, picking up a number of bugfixes and performance improvements to pushing images
+- Updated vendored containers/storage library to v1.10, picking up a number of bugfixes
+- Work on the remote Podman client for interacting with Podman remotely over Varlink is progressing steadily, and many image and pod commands are supported
+- Added path masking to mounts with the `:z` and `:Z` options, preventing users from accidentally performing an SELinux relabel of their entire home directory
+- The `podman container runlabel` command will not pull an image if it does not contain the requested label
+- Many commands' usage information now includes examples
+- `podman rm` can now delete containers in containers/storage, which can be used to resolve some situations where Podman fails to remove a container
+- The `podman search` command now searches multiple registries in parallel for improved performance
+- The `podman build` command now defaults `--pull-always` to true
+
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: Did any of the remote work make it into 1.1?  I think it should be mentioned, even if it's just as a WIP announcement.

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,68 @@
...
+- Fixed a bug where invalid named volumes specified in `podman create` and `podman run` could cause segfaults ([#2301](https://github.com/containers/libpod/issues/2301))
+- Fixed a bug where the `runtime` field in `libpod.conf` was being ignored. `runtime` is legacy and deprecated, but will continue to be respected for the forseeable future
+- Fixed a bug where `podman login` would sometimes report it logged in successfully when it did not
+- Fixed a bug where `podman pod create` would not error on receiving unused CLI argument
+- Fixed a bug where rootless `podman run` with the `--pod` argument would fail if the pod was stopped
+- Fixed a bug where `podman images` did not print a trailing newline when not invoked on a TTY ([#2388](https://github.com/containers/libpod/issues/2388))
+- Fixed a bug where the `--runtime` option was sometimes not overriding `libpod.conf`
+- Fixed a bug where `podman pull` and `podman runlabel` would sometimes exit with 0 when they should have exited with an error ([#2405](https://github.com/containers/libpod/issues/2405))
+- Fixed a bug where rootless `podman export -o` would fail ([#2381](https://github.com/containers/libpod/issues/2381))
+
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: Love the details, just wish we could cut down on the "Fixed a bug" on every line.  Very repetitive but my noggins not coming up with a better solution.

**Comment by mheon** [comment]
Removing WIP, we're good to merge for the release

File: RELEASE_NOTES.md (hunk 4 above)
Comment (mheon) [reply RELEASE_NOTES.md]: There's a note on it in misc

File: RELEASE_NOTES.md (hunk 3 above)
Comment (mheon) [reply RELEASE_NOTES.md]: There's a note on it in misc

**Pull request merged**